### ISSUE TITLE
module not found: ssh2

### ISSUE THREAD

**Issue by tballmsft** [microsoft/genaiscript#493]
- upgrade to latest version of genaiscript using npm
-  npx geniascript is unhappy:
```
> npx genaiscript
Debugger attached.
Waiting for the debugger to disconnect...
Debugger attached.
Debugger attached.
Waiting for the debugger to disconnect...
node:internal/modules/cjs/loader:1227
  throw err;
  ^

Error: Cannot find module 'ssh2'
Require stack:
- C:\Users\tball\AppData\Roaming\npm\node_modules\genaiscript\built\genaiscript.cjs
    at Module._resolveFilename (node:internal/modules/cjs/loader:1224:15)
    at Module._load (node:internal/modules/cjs/loader:1050:27)
    at Module.require (node:internal/modules/cjs/loader:1310:19)
    at require (node:internal/modules/helpers:179:18)
    at ../../node_modules/docker-modem/lib/ssh.js (C:\Users\tball\AppData\Roaming\npm\node_modules\genaiscript\built\genaiscript.cjs:122667:18)
    at __require (C:\Users\tball\AppData\Roaming\npm\node_modules\genaiscript\built\genaiscript.cjs:12:53)
    at ../../node_modules/docker-modem/lib/modem.js (C:\Users\tball\AppData\Roaming\npm\node_modules\genaiscript\built\genaiscript.cjs:126215:15)
    at __require (C:\Users\tball\AppData\Roaming\npm\node_modules\genaiscript\built\genaiscript.cjs:12:53)
    at ../../node_modules/dockerode/lib/docker.js (C:\Users\tball\AppData\Roaming\npm\node_modules\genaiscript\built\genaiscript.cjs:130986:17)
    at __require (C:\Users\tball\AppData\Roaming\npm\node_modules\genaiscript\built\genaiscript.cjs:12:53)
    at Object.<anonymous> (C:\Users\tball\AppData\Roaming\npm\node_modules\genaiscript\built\genaiscript.cjs:223947:32)
    at Module._compile (node:internal/modules/cjs/loader:1480:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1564:10)
    at Module.load (node:internal/modules/cjs/loader:1287:32)
    at Module._load (node:internal/modules/cjs/loader:1103:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:168:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\tball\\AppData\\Roaming\\npm\\node_modules\\genaiscript\\built\\genaiscript.cjs'
  ]
}

Node.js v22.1.0
Waiting for the debugger to disconnect...
```

**Comment by pelikhan** [comment]
repro: https://github.com/microsoft/genaiscript/actions/runs/9311132360/job/25629812972

**Comment by pelikhan** [comment]
fixed in 1.35.1
